server/README.md

```diff
@@ -96,26 +96,34 @@ Because Heroku requires the app to be in the root, we use subtree to push:
 git subtree push --prefix server heroku master
 ```
 
+Updating schema:
+
```

Comment:
what is an example of a situation where we would want to update the schema?  
Is it like refreshing the database?

Reply:
every time I'm adding a view, a table or changing a stored procedure I'm "updating the schema". 

The schema is the structure of the database.